src/components/Content/Login/index.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import LoginForm from "./LoginForm/LoginForm";
+
+const Login = () => {
+  return (
+    <div className="posts">
```

Comment:
Should it be something like that?
```suggestion
    <div className="login">
```